### ISSUE TITLE
Reddit comments

### ISSUE THREAD

**Issue by deniszholob** [deniszholob/factorio-cheat-sheet#180]
Ref: https://www.reddit.com/r/factorio/comments/c9xc0a/factorio_cheat_sheet_update_july_19/

* ~~Add the train speed calculators under community links. https://calculatorio.com/train_acceleration/~~
* ~~Add macOS Keyboard Shortcut differences (See Reddit comment for details)~~
* ~~Add section on rich text shortcut: `To quickly get a rich text tag for an item: open the console, then shift-click on the item (eg in your inventory, a chest, etc.) A rich text tag for that item appears in the console and can be copied.`~~
* ~~Material processing: Put the "round up" asterisk on the rounded numbers not the entire table?~~
* ~~Uranium facts: Asterisk text does not mention that 1 per 10 ore is an average (you can get 0 or 2 very rarely)~~
* ~~Solar power: Should state to add extra accumulators for burst consumption (which would cause the night consumption to exceed average usage)~~
* ~~Weaved -> Woven (https://writingexplained.org/weaved-or-woven-correct)~~
* Nuclear power: Table for power should include a 2xN row showing the theoretical cap of the design, which is 400% efficiency, with the numbers in terms of N. 
| Count: N | Config: 2x(N/2) | Pumps: 1.37N | Exchanger: 1.58N | Turbine: 2.72N | Tanks: 1.31N | Power: 160N MW | Efficiency: 400% |
So basically the reverse calculation for N from 400% efficiency as the last entry.
* ~~Nuke Turbines formula 5.8MW => 5.82MW~~

**Comment by HonkingGoose** [comment]
Hi @deniszholob I've added some pull requests for you to review. I don't really understand the remaining issues on the list, so you'll have to fix that yourself. :smile:

**Comment by deniszholob** [comment]
@HonkingGoose Thanks for jumping on this, I would request all your changes be all in 1 PR however. (You can have different commits if you like)

**Comment by deniszholob** [comment]
Not going to worry about the 400% nuke thing, cool fact, but not really needed. Most people gunna stick with a 4x4 or like 8 or 10 reactors max, gets too crazy after unless you tile and then it doesnt matter either cuz ur going to add more tiles when you need to
